app/views/layouts/application.html.erb

```diff
@@ -9,11 +9,44 @@
   </head>
 
   <body>
+    <% if session[:user_id] != nil %>
+      <% user = User.find(session[:user_id]) %>
+    <% end %>
     <nav class = "topnav">
```

Comment:
I feel like all of the links in this nav bar can be DRY'd up. I'm just unsure how right now. Maybe with a partial?